data.sql

```diff
@@ -6,3 +6,13 @@ VALUES
 ('Gabumon', '2018-11-15', 2, true, 08),
 ('Pikachu', '2021-01-07', 1, false, 15.04),
 ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES
+('Charmander', '2021-11-15', 0, false, -11),
+('Plantmon', '2020-02-08', 2, true, -5.7),
+('Squirtle', '1993-04-02', 3, false, -12.13),
+('Angemon', '2005-06-12', 1, true, -45),
+('Boarmon', '2005-06-07', 7, true, 24.4),
```

Comment:
> Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.

- [ ] According to the requirement, the weight of `Boarmon` is 20.4kg, currently, your data is `24.4kg`. I'll suggest you update this accordingly. :smile: :wave:

Reply:
- Executed below statements to resolve bug: 
<pre>
`BEGIN TRANSACTION;
UPDATE animals SET weight_kg = 20.4 WHERE name = 'Boarmon';
SELECT * FROM animals;
COMMIT;`
<pre>